device_calendar/lib/src/common/constants.dart

```diff
@@ -2,5 +2,6 @@ class Constants {
   static const String fromJsonMapIsNull = 'The json object is null';
 
   static const String invalidArgument = 'Provided arguments are invalid';
-  static const String createOrUpdateEventArgumentRequirements = 'To create or update an event you must provide calendar id, event with title, start date and end date (where start date must be before end date)';
+  static const String createOrUpdateEventArgumentRequirements =
```

Comment:
Is this a correct formatting?

Reply:
yes via flutter format

Reply:
Sorry, I thought there's an extra line space between the variable name and its value.

You should be careful with flutter format though, as sometimes it breaks the logic of the app - situation that I had with chaining extension methods